src/urdf-parser.cpp

```diff
@@ -24,8 +24,6 @@
 #include <QStringList>
 #include <QtGlobal>
 
-#include <osgQt/Version>
```

Comment:
Do you have a problem with this ?
With OSG 3.4, this includes checks that current Qt version matches the one of osgQt.

Reply:
> remove an inclusion of osgQt/Version, not available in OSG 3.2.1

Alright, understood. Can you surround it with OSG_VERSION_GREATER_THAN ?

Reply:
Will do

Reply:
Do you have any idea of how we can know the Qt version used by osgQt ?
ldd can provide the answer but it's very ugly...

Reply:
I think I know the answer.
`osgQt/Version` was added when they started to support Qt 4 and Qt 5. If not present, I guess it is safe to assume that osgQt uses Qt 4.